internal/workflow/preprocessing.go

```diff
@@ -219,7 +219,11 @@ func (w *PreprocessingWorkflow) Execute(
 	}
 
 	if validateMetadata.Failures != nil {
-		result.validationError(ctx, ev, "metadata validation has failed", validateMetadata.Failures)
+		escapedFailures := make([]string, len(validateMetadata.Failures))
+		for _, f := range validateMetadata.Failures {
+			escapedFailures = append(escapedFailures, strings.ReplaceAll(f, identifySIP.SIP.Path, ""))
```

Comment:
Because you've already allocated the elements of the `escapedFailures` slice, you don't want to use append here as it will make the slice length = len(validateMetadata.Failures) * 2. E.g. https://go.dev/play/p/hWlL9IMZ9sA.

Instead you can do:
```
for i, f := range validateMetadata.Failures {
	escapedFailures[i] = strings.ReplaceAll(f, identifySIP.SIP.Path, "")
}
```

Reply:
Makes sense. Now that I think about it, how do you feel about this? @djjuhasz 
```go
		for idx, f := range validateMetadata.Failures {
			validateMetadata.Failures[idx] = strings.ReplaceAll(f, identifySIP.SIP.Path, "")
		}
		result.validationError(ctx, ev, "metadata validation has failed", validateMetadata.Failures)
```

Reply:
Now new slice.

Reply:
Done 8ec4c63f8c8449e425f955b9602aa912b47a2fe5